crates/bootstrap_srv/src/test.rs

```diff
@@ -849,9 +849,6 @@ fn expiration_prune() {
     assert_eq!(1, res.len());
 }
 
-/// Certificates created with:
-/// > openssl genrsa 2048 > certs/test_key.pem
-/// > openssl req -x509 -days 1000 -new -key certs/test_key.pem -out certs/test_cert.pem
```

Comment:
I think keeping this comment could still be valuable

Reply:
I originally did generate certificates for the test and added this comment. As part of the review we switched to generating the certificates in the test with `rcgen`, so this comment should have been tidied up then